modules/data-entry/src/main/frontend/src/questionnaire/MultipleChoice.jsx

```diff
@@ -571,6 +573,9 @@ function ResponseChild(props) {
                   label: classes.inputLabel
                 }}
               />
+              <br/>
+              <MDEditor.Markdown className={classes.selectionDescription} source={description} />
```

Comment:
To ensure consistency with the [section description](https://github.com/data-team-uhn/cards/blob/ed1019d0857eac1ea903a3362f77d8d137b317aa/modules/data-entry/src/main/frontend/src/questionnaire/Section.jsx#L80), we should wrap this in a caption typography and inherit its fontsize:
```suggestion
              <Typography variant="caption" component="div" color="textSecondaty">
                <MDEditor.Markdown className={classes.selectionDescription} source={description} />
              </Typography>
```

Reply:
Going a bit further, I'm not a big fan of seeing a component with "Editor" in its name being used to **display** text.

I suggest creating a separate component `FormattedText` to be used as follows:
```
<FormattedText variant={any valid Typography varian; optional} color={any valid Typography color; optional}>
  {the text to render}
</FormattedText>
```
For example, in the case of our descriptions displayed in the form:
```
<FormattedText variant="caption" color="textSecondary">
  {description}
</FormattedText>
```

Internally, the component would render:

```
<Typography variant={variant} component="div" color={color}
    <MDEditor.Markdown className={classes.formattedText} source={children} />
</Typography>
```
and would incorporate the styling for `classes.formattedText` to ensure the `fontSize` is `inherit`ed.

Reply:
Why do you want to pass text as a node and not as a another prop?

Reply:
I find the usage of the component more logical with the text inside than as a prop, the way it is for Typography for example.

I'm also hoping have a broader use for this component, for example for formatted messages to the user, e.g. like this (completely made up, with excessive formatting just to serve as an example):

```
<Alert severity="error">
  <FormattedText>
      You cannot create more than **eight** forms for this subject. If you wish to add more information of this type:
      * delete an existing form before creating a new one, *or*
      * modify an existing form
  </FormattedText>
</Alert>
```
Using markdown to specify formatting in such messages will be much easier for the dev than using html markup and styling. But when we have long texts on multiple lines, they're easier to read and write as a child of the component than as a prop.